### DISCUSSION TITLE
Example with `alias`?

### DISCUSSION THREAD

**Discussion by wstevick** [tree-sitter/tree-sitter#2616] [Q&A]
I'm learning tree-sitter, and I'm confused about how to use the `alias` function. Where can I see a real-world example of it being used?

Thank you.

**Comment by sogaiu** [comment]
I did a search over a sizable collection and about 2 out of 3 grammars seem to use `alias` (that's based on hits in 214 out of 308 grammars).

You can see lists of grammars in at least the following two locations:

* https://tree-sitter.github.io/tree-sitter/#parsers
* https://github.com/sogaiu/ts-questions/blob/master/ts-grammar-repositories.txt

**Comment by wstevick** [comment]
Thanks.
